doc/tools/west/index.rst

```diff
@@ -57,17 +62,38 @@ West was introduced to unify and streamline the developer's experience when
 working with Zephyr. It is very important to note that west does not
 replace the CMake-based build system, but directly uses it and augments it.
 
+See :ref:`west-flash-debug` for a detailed explanation of the flash and debug
+command-line interface exposed by west.
+
 .. _west-struct:
 
 Structure
 *********
 
+West structure
+==============
+
 West is downloaded and installed on a system in two stages:
 
 * Bootstrapper: Installed using ``pip``, implements ``west init``
 * Installation: Installed using ``west init``, implements all other commands
 
-Additional information about the two parts can be found below.
+Additional information about the two distinct parts of west can be found in
+the corresponding sections below.
+
+Repository structure
+====================
+
+Beyond west itself, the actual code repositories that west works with
+are the following:
+
+* Manifest repository: Cloned by ``west init``, and managed afterward with Git
+  by the user. Contains the list of projects in the manifest :file:`west.yml`
```

Comment:
How about re-phrasing this:
```Contains the list of projects in the manifest :file:`west.yml` file.```
into
```The file :file:`west.yml` contains a list of additional projects used by Zephyr.```

Reply:
OK, will change this one.

Reply:
Actually on second thought I'd rather not change this one. This describes a generic manifest repo, not the zephyr one and the current wording is clearer in that case.

Reply:
then maybe just:
```The file :file:`west.yml` contains a list of additional projects to clone.```